.github/workflows/cypress.yaml

```
@@ -81,58 +76,17 @@
-          parallel: true
-          group: "UI-Chrome"
-        env:
-          CYPRESS_SPLIT_TESTS: "true"
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NODE_OPTIONS: --max_old_space_size=4096
-          COMMIT_INFO_MESSAGE: ${{github.event.pull_request.title}}
-          COMMIT_INFO_SHA: ${{github.event.pull_request.head.sha}}
-          SPLIT: ${{ strategy.job-total }}
-          SPLIT_INDEX: ${{ strategy.job-index }}
+      - name: Run Cypress Tests in Parallel 🥬
+         run: |
```

actionlint (1.7.3)

could not parse as YAML: yaml: line 80: mapping values are not allowed in this context

(syntax-check)